Add get-function for the timeout member variable.

cstim opened the pull request:
In C++ it is good practice to have a get-function alongside each
set-function. In this particular case e.g. debug output might want
to return the actual timeout value of this particular timer object.

alextaujenis commented:
I agree getTimeout() should be included with this library. I like how short your commit is (a one-liner), however I'd like to keep all application logic inside of the source .cpp file. Can you move the function logic to the source .cpp file, and also write a passing Unit Test? Maybe something like this:

``` arduino
// getTimeout
  test(getTimeout_should_return_the_timeout_in_milliseconds) {
    // fill in test logic here
  }
```

cstim committed:
Add get-function for the timeout member variable.

In C++ it is good practice to have a get-function alongside each
set-function. In this particular case e.g. debug output might want
to return the actual timeout value of this particular timer object.

Also, a unittest for this accessor is added as well.

cstim commented:
will create a new pull request in a second
